index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <title>Profolio</title>
+</head>
+<body>
+
+        <div class="container">
+
+            <section class="Header">
+                
+                    <ul class="header-logo">
+                        <li class="logo"> My Logo</li>
+                        <li class="hamburg-logo"> <img src="Img/Hlogo.png" alt=""></li>
+                
+                    </ul>
+
+                    <div class="section-header">
+
```

Comment:
Kindly make use of the `section` tag instead of the `div` tag in accordance with HTML semantic tag usage. You can read more about it [here](https://www.freecodecamp.org/news/semantic-html5-elements/) 😊